bqetl_project.yaml

```diff
@@ -385,3 +385,8 @@ render:
   skip:
   # uses {%s} which results in unknown tag exception
   - sql/mozfun/hist/string_to_json/udf.sql
+
+glean_usage:
+  override: # Skip automatically updating the following artifacts
+    skip:
```

Comment:
Rather than `override: skip:` I'd suggest `skip_existing:`, as I think the current structure can be misread as the skips being overrides rather than skipping overriding/overwriting (the comment does help, but IMO inherent names are the clearest documentation).